client/core/core.go

```diff
@@ -4203,17 +4206,31 @@ func (c *Core) prepareTrackedTrade(dc *dexConnection, form *TradeForm, crypter e
 		if len(pubKeys) == 0 || len(sigs) == 0 {
 			return nil, 0, newError(signatureErr, "wrong number of pubkeys or signatures, %d & %d", len(pubKeys), len(sigs))
 		}
-		redemptionReserves, err = accountRedeemer.ReserveN(redemptionLots, wallets.toAsset.MaxFeeRate, wallets.toAsset.Version)
+		redemptionReserves, err = accountRedeemer.ReserveNRedemptions(redemptionRefundLots, wallets.toAsset.MaxFeeRate, wallets.toAsset.Version)
```

Comment:
Works for me.  Only additional thought about this matter is that the rate given to `ReserveNRedemptions` and `ReserveNRefunds` could be the larger of `MaxFeeRate` and the rate from `FeeRate` (if also a `FeeRater`).  That would be pretty screwy if the server had a rate set that low, so whatever, let's not change anything now.  This PR looks good to me.

Reply:
If the swap cannot be initiated, then it doesn't matter if too little is reserved for refunds and redemptions.

Reply:
The swap tx could still be mined with a fee less than what FeeRate would recommend.